pymc3/distributions/continuous.py

```diff
@@ -361,8 +361,8 @@ class Flat(Continuous):
     rv_op = flat
 
     @classmethod
-    def dist(cls, *, size=None, initval=None, **kwargs):
-        if initval is None:
+    def dist(cls, *, size=None, initval=UNSET, **kwargs):
+        if initval == UNSET or initval is None:
```

Comment:
These two distributions have a different behavior because they cannot be sampled from.
For consistency we might want to consider raising an error on `initval=None` instead?

Reply:
The error will be raised anyway during sampling of the initial point. I don't see anyone setting a Flat initial value to None and not understanding where the error is coming from

Reply:
I implemented an error in the `make_initval` method to make it easier to spot. Feel free to review/comment the updated code again :)